include/vertices_types.h

```diff
@@ -46,12 +67,36 @@ typedef struct
 
 typedef struct
 {
-    char public_b32[PUBLIC_B32_STR_MAX_LENGTH]; // b64 public address, with \0
-    unsigned char private_key[ADDRESS_LENGTH]; // 32-bytes private key
-    unsigned char public_key[ADDRESS_LENGTH]; // 32-bytes public key
-    int32_t amount; // tokens on account
+    char public_b32[PUBLIC_B32_STR_MAX_LENGTH]; //!< b32 public address, with `\0` termination character
+    unsigned char private_key[ADDRESS_LENGTH];  //!< 32-bytes private key
+    unsigned char public_key[ADDRESS_LENGTH];   //!< 32-bytes public key
+    int32_t amount;                             //!< amount of tokens on account
 } account_info_t;
 
+typedef enum
+{
+    VALUE_TYPE_NONE = 0,
+    VALUE_TYPE_BYTESLICE,
+    VALUE_TYPE_INTEGER,
+} value_type_t;
```

Comment:
We also need to support **address** and **base64** types (cf. algoDEA).

For instance:
```c
typedef enum
{
    VALUE_TYPE_NONE = 0,
    VALUE_TYPE_BYTESLICE,
    VALUE_TYPE_INTEGER,
    VALUE_TYPE_ADDRESS,
    VALUE_TYPE_BASE64,
} value_type_t;
```